src/qt/utilitydialog.cpp

```diff
@@ -33,29 +33,46 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
     ui(new Ui::HelpMessageDialog)
 {
     ui->setupUi(this);
-
+    this->resize(500,0);//force view height to stretch
```

Comment:
The Ui::HelpMessageDialog is shared with the about view.
These changes force the view to stretch to an appropriate height -  this eliminates excessive height dimensions - the user resizes based upon personal preference.